Codigo/modNetwork.bas

```diff
@@ -162,19 +169,21 @@ On Error GoTo OnServerClose_Err:
 114    End If
     
 116    Debug.Assert IsValidUserRef(UserRef)
-118    If Not IsValidUserRef(UserRef) Then Exit Sub
-    
-120    If UserList(UserRef.ArrayIndex).flags.UserLogged Then
-122        Call CloseSocketSL(UserRef.ArrayIndex)
-124        Call Cerrar_Usuario(UserRef.ArrayIndex)
-126    Else
-128        Call CloseSocket(UserRef.ArrayIndex)
-130    End If
+118    If IsValidUserRef(UserRef) Then
+120        If UserList(UserRef.ArrayIndex).flags.UserLogged Then
+122            Call CloseSocketSL(UserRef.ArrayIndex)
+124            Call Cerrar_Usuario(UserRef.ArrayIndex)
+126        Else
+128            Call CloseSocket(UserRef.ArrayIndex)
```

Comment:
call LogSecurity("CloseSocket not valid user ref")

Reply:
why log security?
its one of the few logs that I cant request from remote and seems to be oriented for cheats log

trace error does the intended work and can be requested by /reqdebug command